opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -629,14 +632,14 @@ class Well {
     double solvent_fraction;
     bool has_produced = false;
     bool prediction_mode = true;
-
+    double productivity_index{ std::numeric_limits<double>::lowest() };
```

Comment:
operator==, serializeOp, serializeObject ?

Reply:
It is indeed missing from `operator==`.  Thanks.  I do however not understand what you mean by the `serializeObject` comment.  The member already has an initializer whence the default constructor should do everything we need.  What am I missing?

Reply:
a default value does not really test anything.

Reply:
> a default value does not really test anything.

Right, so the `serializeObject()` function exists only to set up a non-defaulted object for testing the serialization procedure.  In that case, I think it should have another name&mdash;e.g., `serializationTestObject()`.

Reply:
I think we need to create a abstract class interface called ThisIsAnAbstractClassInterfaceForASerializationTestObjectCreationMethod, and then construct a factory for these objects.